Resources/Prototypes/_White/Entities/Objects/Specific/Robotics/airemote.yml

```
@@ -0,0 +1,15 @@
+  name: "B.O.R.I.S. AI-Cyborg Remote Control"
+  description: Artificial Intelligence cyborg control board.
+  components:
+    - type: Sprite
+      sprite: Objects/Misc/module.rsi
+      state: boris
+    - type: Examiner
+    - type: AiRemoteBrain
+    - type: Actions
+    - type: Tag
+      tags:
+      - CannotSuicide
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)